sdk/spring/spring-cloud-azure-feature-management/src/test/java/com/azure/spring/cloud/feature/management/filters/TargetingFilterTest.java

```diff
@@ -23,9 +23,9 @@
 @SpringBootTest(classes = { TestConfiguration.class, SpringBootTest.class })
 public class TargetingFilterTest {
 
-    private static final String USERS = "users";
+    private static final String USERS = "Users";
```

Comment:
Please add a test for lowercase "users" too to ensure backward compatibility.

Reply:
Added